ts/indicators/bollinger-bands.src.ts

```diff
@@ -196,13 +196,13 @@ H.seriesType<Highcharts.BBIndicator>(
         },
         getValues: function (
             this: Highcharts.BBIndicator,
-            series: Highcharts.BBIndicator,
+            series: Highcharts.Series,
             params: Highcharts.BBIndicatorParamsOptions
         ): (boolean|Highcharts.IndicatorMultipleValuesObject) {
             var period: number = (params.period as any),
                 standardDeviation: number = (params.standardDeviation as any),
                 xVal: Array<number> = (series.xData as any),
-                yVal: Array<Array<number>> = series.yData,
+                yVal: Array<Array<number>> = (series.yData as any),
```

Comment:
Additionally the `BBIndicator.yData` property should be removed in the class declaration.